Index: Back-End/SpiritPleasure/SpiritPleasure/urls.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.contrib import admin\r\nfrom django.urls import path\r\n\r\nurlpatterns = [\r\n    path('admin/', admin.site.urls),\r\n]\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Back-End/SpiritPleasure/SpiritPleasure/urls.py b/Back-End/SpiritPleasure/SpiritPleasure/urls.py
--- a/Back-End/SpiritPleasure/SpiritPleasure/urls.py	
+++ b/Back-End/SpiritPleasure/SpiritPleasure/urls.py	
@@ -1,6 +1,7 @@
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('authentication.urls'))
 ]
Index: Back-End/SpiritPleasure/authentication/urls.py
===================================================================
diff --git a/Back-End/SpiritPleasure/authentication/urls.py b/Back-End/SpiritPleasure/authentication/urls.py
new file mode 100644
--- /dev/null	
+++ b/Back-End/SpiritPleasure/authentication/urls.py	
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('register/', views.AuthenticationView.as_view(), name='create_user')
+]
